### PR TITLE
Add replug tests for round-trip case and no portDown case

### PR DESCRIPTION
**Pull request by rtretyak**

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy [hunk 1]
```diff
@@ -21,11 +24,63 @@ import spock.lang.Narrative
...
+            it.aswitch?.inPort && it.aswitch?.outPort &&
+                    it.srcSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD) &&
+                    it.dstSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)
+        }
+        assumeTrue("Wasn't able to find enough of required a-switch links with round-trip", isl.asBoolean())
+
+        and: "A non-connected a-switch link"
+        def notConnectedIsl = topology.notConnectedIsls.find {
+            it.srcSwitch != isl.srcSwitch &&
+                    it.srcSwitch.features.contains(SwitchFeature.NOVIFLOW_COPY_FIELD)
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy:45]: Why do you need `NOVIFLOW_COPY_FIELD`? Do you need support RTL for the non-connected link?
If so, let's add this info into the step description

Comment (rtretyak) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy:45]: changed to `and: "A non-connected a-switch link with round-trip support"`

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy [hunk 2]
```diff
@@ -21,11 +24,63 @@ import spock.lang.Narrative
...
+        assert northbound.deleteLink(islUtils.toLinkParameters(newIsl)).size() == 2
+
+        then: "Moved ISL is removed"
+        Wrappers.wait(WAIT_OFFSET) {
+            assert !islUtils.getIslInfo(newIsl).isPresent()
+            assert !islUtils.getIslInfo(newIsl.reversed).isPresent()
+        }
+
+        and: "The src and dst switches of the isl pass switch validation"
+        [isl.srcSwitch.dpId, isl.dstSwitch.dpId].each { swId ->
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy:77]: How about validating `notConnectedIsl.srcSwitch` ?

Comment (rtretyak) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/IslReplugSpec.groovy]: added

**Pull request merged**